### PR TITLE
Ingestion Recommendation for Neural Search

### PR DESCRIPTION
**Pull request by vibrantvarun**
### Description
Recommendation for bulk size ingestion for neural Search.

### Issues Resolved
Fixes [5175](https://github.com/opensearch-project/documentation-website/issues/5175)

### Checklist
- [ ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by hdhalter** [comment]
@kolchfa-aws - Do you want to review this one, Fanit? Thanks.

File: _search-plugins/neural-text-search.md [hunk 1]
```diff
@@ -110,6 +110,9 @@ PUT /my-nlp-index/_doc/2
 
 Before the document is ingested into the index, the ingest pipeline runs the `text_embedding` processor on the document, generating text embeddings for the `passage_text` field. The indexed document includes the `passage_text` field, which contains the original text, and the `passage_embedding` field, which contains the vector embeddings. 
 
+### Ingestion Recommendation
+The throughput for ingestion varies with the bulk size. Neural search plugin supports different remote connectors for inference like SageMaker, Amazon Bedrock, OpenAI etc. We recommend to have bulk size up to 500 docs for SageMaker and for all other connectors we recommend to refer the API limits for bulk size to get the best throughput.
```
Comment (kolchfa-aws) [inline _search-plugins/neural-text-search.md:114]:
```suggestion
Ingestion throughput varies with bulk size. Neural search supports different [connectors]({{site.url}}{{site.baseurl}}/ml-commons-plugin/remote-models/connectors/) for models hosted on external platforms, such as SageMaker, Amazon Bedrock, and OpenAI. For SageMaker, we recommend bulk size up to 500 documents. For all other connectors, we recommend to refer to the API limits for bulk size for best throughput. For more information, see [Bulk API]({{site.url}}{{site.baseurl}}/api-reference/document-apis/bulk/).
{: .important}

```

Comment (kolchfa-aws) [reply _search-plugins/neural-text-search.md:114]: I don't think we have any limits discussed on the [Bulk API page](https://opensearch.org/docs/latest/api-reference/document-apis/bulk/). Should we add them?

**Review by kolchfa-aws — COMMENTED** [review]
Thank you, @vibrantvarun! What version is this applicable to?

**Pull request closed without merging**